spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  before do
+    @user  = FactoryBot.create(:user)
+    @post = FactoryBot.create(:post)
+    @comment = FactoryBot.create(:comment)
+  end
+
+  describe '#create' do
```

Comment:
In `create` or `update` method you should have 2 context: `with valid params` and `with invalid params`